example-get-started-experiments/generate.sh

```diff
@@ -106,15 +106,15 @@ dvc stage add -n train \
   -p base,train \
   -d src/train.py -d data/train_data \
   -o models/model.pkl \
-  -M results/train/metrics.json \
-  --plots-no-cache results/train/plots \
+  -O results/train/metrics.json \
+  -O results/train/plots \
```

Comment:
Also happy to drop these completely, which would simplify a lot of output, but want to coordinate this between here, https://github.com/iterative/dvc.org/pull/4420, and https://github.com/iterative/dvclive/pull/497.

Reply:
I vote in favor of going all-in with  the dropping approach across all places. 

We still need to find a place/way to explain how the UIs are finding the stuff to be displayed in this scenario, though

Reply:
Updated to drop them.

Where are you thinking to explain them? We have https://dvc.org/doc/dvclive/live/make_dvcyaml, and we could maybe reference it in https://dvc.org/doc/dvclive/how-it-works#track-the-results. Do you think we need it anywhere else?